e2e/main_test.go

```diff
@@ -41,5 +43,16 @@ func TestMain(m *testing.M) {
 	hasExperimental = bytes.Contains(output, []byte("Experimental: on"))
 	i := strings.Index(string(output), "Renderers")
 	renderers = string(output)[i+10:]
+
+	setupContext()
+
+	icmd.RunCommand("docker", "swarm", "init") // Required by "docker-app install"
 	os.Exit(m.Run())
+	icmd.RunCommand("docker", "swarm", "leave", "--force") // Unique manager requires "--force" to leave
+}
+
+func setupContext() {
+	icmd.RunCommand(dockerApp, "add-credentialset", "--force", credsName, targetName)
```

Comment:
These should be removed as the context will be handled by the docker CLI itself. Maybe it can be moved to the Dockerfile while creating the e2e image?

Reply:
I agree with the first statement, but for the second the problem is that by moving this to the Dockerfile would not help when running the tests in the host machine (with just `make test-e2e`), which is my case on GNU/Linux for example :smiley:

Reply:
So maybe let make test-e2e handle that? 😉

Reply:
Done